examples/basic/MODULE.bazel

```diff
@@ -0,0 +1,9 @@
+PYBIND_VER="2.12.0"
+bazel_dep(name = "pybind11_bazel", version = "%s" % PYBIND_VER)
+bazel_dep(name = "rules_python", version = "0.33.1")
+
+python = use_extension("@rules_python//python/extensions:python.bzl", "python")
+python.toolchain(
+    configure_coverage_tool = True,
+    python_version = "3.9" ,
+)
```

Comment:
Leave a comment to explain why this is being done explicitly? Otherwise, you can just omit it and then rules_python will use whatever its default is.